osf/models/institution.py

```diff
@@ -8,19 +8,43 @@
 
 
 class Institution(Loggable, base.ObjectIDMixin, base.BaseModel):
+
     # TODO Remove null=True for things that shouldn't be nullable
-    banner_name = models.CharField(max_length=255, null=True, blank=True)
+    # e.g. CharFields should never be null=True
+
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True)
```

Comment:
Yes, better to be explicit. Since `null=True`, then `description` will default to `None`. Better to keep the data consistent and keep the default as `''`.